.php_cs

```diff
@@ -107,10 +153,15 @@ $config->setRules(
         'single_line_comment_style' => true,
         'single_quote' => true,
         'space_after_semicolon' => true,
+        'standardize_increment' => true,
         'standardize_not_equals' => true,
+        'strict_comparison' => true,
+        'strict_param' => true,
+        'string_line_ending' => true,
         'ternary_operator_spaces' => true,
         'ternary_to_null_coalescing' => true,
         'trailing_comma_in_multiline_array' => true,
+        'trim_array_spaces' => true,
```

Comment:
👎 Personally, I like spaces in single line arrays.

Reply:
Hmm. That's.... fair.